frontend/src/components/DashboardHeader.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
+    <v-app-bar class="navbar-custom" dark clipped-right>
+      <v-toolbar-title>
+        <a href="https://codeforpoznan.pl/">
+          <v-img
+            :src="image"
+            contain
+            max-height="20"
+            aspect-ratio="1.7"
+            max-width="500"
+            min-width="350"
```

Comment:
Some older phones have 320px displays. E.g. I checked iPhone 4 and it fits all right but doesn't leave the space for hamburger icon if we decide to use one. I suggest set the min-width to 300px

Reply:
@OtisRed Do You thing it is still valid? I wasn't able to reproduce that. Hamburger menu is in place and working on 300px screen, changing min-width to suggested 300px, have no impact. Apart from that it is related to header, not to login/logout with refresh logic, which is these pr feature.